### PR TITLE
[AF-1555]: Upgrade WF11 to WF14

### PR DESCRIPTION
**Pull request by adrielparedes**
## Description

Upgrate to Wildfly 14

## Related to:
* https://github.com/jboss-integration/jboss-integration-platform-bom/pull/427
* https://github.com/errai/errai/pull/363
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/843
* https://github.com/kiegroup/appformer/pull/483
* https://github.com/kiegroup/kie-wb-common/pull/2147
* https://github.com/kiegroup/jbpm/pull/1336
* https://github.com/kiegroup/jbpm-wb/pull/1206
* https://github.com/kiegroup/drools/pull/2086
* https://github.com/kiegroup/droolsjbpm-integration/pull/1583
* https://github.com/kiegroup/kie-wb-distributions/pull/821 
* https://github.com/kiegroup/kie-docs/pull/1108
* https://github.com/kiegroup/optaplanner/pull/437
* https://github.com/kiegroup/optaplanner-wb/pull/304

### CODE REVIEW

**Comment by mswiderski** [comment]
you're missing these changes https://github.com/mswiderski/kie-wb-distributions/commit/695b1893ae97d330e290e7df166c85d547e805fb

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: kie-server-distributions/jbpm-server-distribution/src/main/server/standalone.xml [hunk 1]
```diff
@@ -159,17 +161,6 @@
                         <password>sa</password>
                     </security>
                 </datasource>
-                <xa-datasource jndi-name="java:jboss/datasources/jBPMDS" pool-name="jBPMXADS" enabled="true">
```
Comment (cristianonicolai) [inline kie-server-distributions/jbpm-server-distribution/src/main/server/standalone.xml]: @adrielparedes should this be removed?

File: kie-server-distributions/jbpm-server-distribution/src/main/server/standalone.xml [hunk 2]
```diff
@@ -482,20 +439,14 @@
                         <login-module code="RealmDirect" flag="required">
                             <module-option name="password-stacking" value="useFirstPass"/>
                         </login-module>
-                        <login-module code="org.kie.security.jaas.KieLoginModule" flag="optional" module="deployment.jbpm-console.war"/>
```
Comment (cristianonicolai) [inline kie-server-distributions/jbpm-server-distribution/src/main/server/standalone.xml]: @adrielparedes similar comment here, shouldn't this stay as is?

**Comment by cristianonicolai** [comment]
@adrielparedes @ederign I created a PR with necessary fixes for the standalone.xml, please incorporate changes before merging this. See: https://github.com/adrielparedes/kie-wb-distributions/pull/1

**Comment by adrielparedes** [comment]
@cristianonicolai thanks for the PR. I've integrated it with AF-1555

**Pull request merged**